### PR TITLE
Show claimed context when already claimed

### PR DESCRIPTION
**Pull request by alfetopito**
# Summary

Continues work started by Nenad adding a bit more context when claim is already claimed

Screen Shot 2022-02-09 at 13 57 44

  # To Test

1. Claim everything with one account
* You should see the amount of vCOW claimed for that account
* You should see an updated body and button text as well
2. Check other states (account with no claims and account with claims)
* Should remain as they where

### CODE REVIEW

**Comment by alfetopito** [comment]
Feels a bit redundant (`already claimed` text both on the banner and the body). Any suggestions to improve the text?

**Comment by elena-zh** [comment]
Hey @alfetopito , nice changes!

> Feels a bit redundant (already claimed text both on the banner and the body). Any suggestions to improve the text?

I'd suggest to show only one message about 'already claimed' , and show it in the warning at the top of the form. Maybe, I'd add a network indicator into the warning
claimed

Also, 1 tiny nitpick: we do not replace 'Connect wallet' button with a disabled button when check an info for an account being not connected. 
Maybe we should follow the same logic when displaying 'already claim' screen? WDYT?
change button
connect a wallet

File: src/custom/pages/Claim/ClaimSummary.tsx [hunk 1]
```diff
@@ -7,23 +7,31 @@ import { ClaimSummary as ClaimSummaryWrapper, ClaimSummaryTitle, ClaimTotal } fr
 import { ClaimCommonTypes } from './types'
 import { ClaimStatus } from 'state/claim/actions'
 import { AMOUNT_PRECISION } from 'constants/index'
+import { useTokenBalance } from '@src/state/wallet/hooks'
```
Comment (nenadV91) [inline src/custom/pages/Claim/ClaimSummary.tsx:10]: Nitpick, can we remove `@src` here

**Review by nenadV91 — APPROVED** [review]

**Comment by elena-zh** [comment]
@alfetopito , this might be not related to the current PR, but I noticed that when I leave a PR tab opened for several hours, then get back to it, a lot is infura errors appear in the console. See the video: https://watch.screencastify.com/v/TJzMWMd7fJ09paKXSoD8
Could you please clarify, is this OK?

**Comment by alfetopito** [comment]
@elena-zh addressed your suggestions:

## Connected
Screen Shot 2022-02-10 at 09 18 57

## Not connected
Screen Shot 2022-02-10 at 09 19 15

**Comment by elena-zh** [comment]
Great! Thank you!

**Pull request merged**